lib/tasks/impersonation_admin_user.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+namespace :impersonation_admin_user do
+  task :update, %i[username domain] => :environment do |_task, args|
+    new_username = args[:username]
+    new_domain = args[:domain]
+    User.where(username: '3scaleadmin').find_each do |user|
+      puts "Updating user ##{user.id}"
+      user.update!({username: new_username, email: "#{new_username}+#{user.account.self_domain}@#{new_domain}"})
```

Comment:
The email is limited to 255 characters now https://github.com/3scale/system/pull/9413
Remember, we encountered some long domain names with the 100 characters issue. That is fixed.
This will work until the first one fails and it will exit the rake task

Probably make it fail for every user or none.